modules/simple_forms_api/app/controllers/simple_forms_api/v1/uploads_controller.rb

```diff
@@ -36,6 +37,11 @@ def submit
 
         if form_is210966 && icn && first_party?
           handle_210966_authenticated
+        elsif params[:form_number] == '26-4555' && icn
```

Comment:
just a thought for the future, I wonder if this could be done in a more configuration based way, so this class wouldn't have to have knowledge of what 4555 is doing, but 4555 could pass an option to say it is a grant application

Reply:
I think this is a great idea! @pennja something for us to consider in terms of refactoring this controller.